### PR TITLE
Update Neuvector v2.4.3 for Rancher v2.7

### PR DESCRIPTION
**Pull request by selvamt94**

### CODE REVIEW

**Comment by nicholasSUSE** [comment]
Hi @selvamt70  we at the mapps team have reviewed your PR.

Please change a few things:
1. [First Commit ](https://github.com/rancher/charts/pull/2536/commits/8439a5096ab76a950fe4bc25d361aa7c37b18ae2) - Please remove the changes at release.yaml file, they should only appear in the last separate commit
2. [Third Commit](https://github.com/rancher/charts/pull/2536/commits/e7a5435638258f7326fc5237b88c18154a681756) - Please do not remove the previous information on versions at release.yaml file, only append yours.

Also for making reviewing easier, please provide us the image-mirror Pull Request link.

If you want you may use this [PR details](https://github.com/rancher/charts/pull/2546) as a model.

Best regards

**Comment by selvamt94** [comment]
Hi @nicholasSUSE . I have made changes as per your comment. Please review new changes. Thanks.

File: release.yaml [hunk 1]
```diff
@@ -10,3 +10,7 @@ rancher-eks-operator-crd:
   - 102.0.2+up1.2.0
   - 102.0.1+up1.1.5
   - 102.0.0+up1.2.0
+neuvector:
```
Comment (diogoasouza) [inline release.yaml:13]: Hi @selvamt94
Apparently the indentation here is wrong, could you fix that?

Comment (selvamt94) [reply release.yaml:13]: I have updated it. Thanks.

**Review by diogoasouza — DISMISSED** [review]
@selvamt94 LGTM, just fix the conflict with the release.yaml file before merging!

**Review by diogoasouza — APPROVED** [review]

**Pull request merged**